en/flight_controller/cuav_v5_plus.md

```diff
@@ -179,25 +154,38 @@ Specifically this will stop the GPS's compass due to contention, and may also da
 
 Similarly, a digital PM plugged into a analog connector will not work, and may also damage/destroy the power module (longer term).
 
-- *Found:* Batch 01
-- *Fixed:* -
+## Compatibility 
+
+CUAV adopts some differentiated designs and is incompatible with some hardware, which will be described below.
+
+#### GPS not compatible with other devices {#compatibility_gps}
 
+The *Neo v2.0 GPS* recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
+The GPS will not work with other flight controllers.UAVCAN GPS is ideal for avoiding this compatibility.
 
-#### Using JTAG for hardware debugging {#issue_jtag}
+#### Using JTAG for hardware debugging {#compatibility_jtag}
 
 `DSU7` FMU Debug Pin 1 is 5 volts - not the 3.3 volts of the CPU.
 
 Some JTAG use this voltage to set the IO levels when communicating to the target.
 
 For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts of DSM/SBUS/RSSI pin 4 as Pin 1 on the debug connector (`Vtref`).
 
-#### The HV\_PM power module output is unfused {#issue_pm_unfused}
+## Known Issues
+
+The issues below refer to the *batch number* in which they first appear. 
+The batch number is the four-digit production date behind V01 and is displayed on a sticker on the side of the flight controller.
+For example, the serial number Batch V011904((V01 is the number of V5, 1904 is the production date, that is, the batch number).
+
+#### SBUS / DSM / RSSI interface Pin1 unfused {#pin1_unfused}
 
-> **Warning** This is a serious safety issue.
+> **Warning** This is a security issue.
```

Comment:
As above, what is a security issue? I know you probably answered this, but why is this not a safety issue?

Reply:
Changed